docs/snippets/defaults_channel_mapping.mdx

```diff
@@ -17,9 +17,13 @@ default_channel_by_device:
     - social
     - audio
     - web
+    - app
   tv:
     - ctv-bvod
+    - social
+    - audio
     - app
+    - web
```

Comment:
I believe that missing this channels is intentional. We do not want to support all channels for all devices. @bokelley

Reply:
removed web from TV and removed app from PC